ToDo/.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
you should update gitignore in the root folder

Reply:
I'm not sure what exactly I needed to update in gitignore, so I did that:
git rm -r --cached .
git add .
git commit -m "Update gitignore file"